octobot_trading/producers/kline_updater.py

```diff
@@ -82,7 +82,8 @@ async def time_frame_watcher(self, time_frame):
 
     async def resume(self) -> None:
         await super().resume()
-        if not self.is_running and self.tasks:
-            for task in self.tasks:
-                task.cancel()
+        if not self.is_running:
+            if self.tasks:
```

Comment:
this part will be changed in https://github.com/Drakkar-Software/OctoBot-Trading/pull/217 I don't think there is a need to check for self.tasks